src/cli.ts

```diff
@@ -434,8 +449,26 @@ async function test(
     opts.provider && AVAILABLE_PROVIDERS.includes(opts.provider)
       ? opts.provider
       : "kubernetes";
+
+  const configBasePath = path.dirname(testFile);
+  const env = new Environment(new RelativeLoader([configBasePath]));
+  const temmplateContent = fs.readFileSync(testFile).toString();
+  const content = env.renderString(temmplateContent, process.env);
+
+  const testName = getTestNameFromFileName(testFile);
```

Comment:
You should probably reuse [the misc util function](https://github.com/paritytech/zombienet/blob/main/src/utils/misc.ts#L115-L128) instead of making a new one

Reply:
same here.